tests/e2e/utils.js

```diff
@@ -25,6 +26,7 @@ import { elementExists, getTextContent } from './page-utils';
  * @typedef {{ addedBy: string, hasActions: boolean, templateTitle: string }} TemplateTableItem
  */
 
+export const BASE_URL = config.get( 'url' );
 export const DEFAULT_TIMEOUT = 30000;
```

Comment:
When debugging got stung by having default timeout here and not in config file, had this one override what I've passed via command line and test timeouted for me even though I've disabled timeouts.

For the future I'd consider tapping into config/env values for that but it's just for consideration

Reply:
This `DEFAULT_TIMEOUT` is actually not a jest/puppeteer timeout configuration. This timeout is used in specific functions like waiting for canvas and a few others. This won't override a test case timeout or lack thereof.